### ISSUE TITLE
Similar style (optional) live icon & (optional) camera name as Apple Homekit

### ISSUE THREAD

**Issue by conorlap** [dermotduffy/frigate-hass-card#1782]
**[REQUIRED] Describe the feature you'd like:**

I'd love to have the _**option**_ to have a "LIVE" icon appear, in similar style to the Apple Home app per attached screenshot, obviously only displaying when the full live feed is playing (ie. not the still images). Would also be nice to have the separate _**option**_  to display the camera name in same style as screenshot. 

Ideally both of these would be movable per users desire. Eg Top left, bottom left etc.

**[OPTIONAL] Describe alternatives you've considered:**
N/A

**[OPTIONAL] Additional context:**

![image](https://github.com/user-attachments/assets/60f29681-be1e-48c4-bca4-1544c70fff03)

**Comment by dermotduffy** [comment]
This is kind of similar to the existing `status_bar` support, i.e.:

```
status_bar:
  style: overlay
  position: bottom
```

... except fancier looking :-)

I think the differences are (roughly):

 - Support styling on strings (orange around live)
 - "Transparent" overlay

**Comment by conorlap** [comment]
Yeah I was meaning to put in the FR that this request is similar to status bar, I'm personally just not keen on it's current styling. 
Been loving the updates lately, thanks as always for this amazing card!!
